torch/csrc/distributed/c10d/reducer.h

```diff
@@ -219,22 +220,17 @@ class Reducer {
     // std::vector<at::cuda::CUDAEvent> events;
   };
 
-  // This function is called inside `initialize_buckets` and
-  // `finalize_backward`. The function call in `initialize_bucket` creates both
+  // This function is called inside `initialize_buckets`, it initializes both
   // views_in and views_out into the contents tensor for each variable's grad.
   // Views serve as entry points to copy_ each grad's data in/out of the flat
-  // contents tensor. The function call in `finalize_backward` happens only if
-  // DDP communication hook was registered to recrate just views_out with the
-  // result of `future_work`. If called from `finalize_backward`,
-  // `initialize_bucket_views` will not modify `bucket_views_in`. This will keep
-  // `bucket_views_in` referring to replica's contents and
-  // `bucket_view_in.copy_(grad)` call inside reducer's
-  // `mark_variable_ready_dense` will work as expected. Note that before the
+  // contents tensor.
+  void initialize_bucket_views(BucketReplica& replica, at::Tensor& contents);
+
+  // This function is called inside `finalize_backward`, it happens only if
+  // DDP communication hook was registered to recreate just views_out with the
```

Comment:
if you are referring to `bucket_views_out`, let's use the full var name (i.e., replace `views_out` with `bucket_views_out`)